### DISCUSSION TITLE
Hide download button on the standalone gallery page

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#194] [Q&A]
Hy, is it possible to hide the download button in standalone gallery page ?
And i have the  error 404 oops nothing here on the slide show page http://192.168.16.106:8000/#/slideshow/random. 
Thx

**Comment by mgineer85** [comment]
Yes in uisettings there is a switch.

**Reply by laguinch** [reply]
Hy the button in uisettings is disable. It’s ok for the admin/gallery but in the standalone/Gallery the download button is enable.

**Reply by mgineer85** [reply]
Hi, currently it's not distinguished between gallery and standalone gallery.
Focusing currently on version 2 with the video feature. Feel free to create an issue for missing functions or send pull requests so it can be added later.
